google/api_core/client_logging.py

```diff
@@ -15,12 +15,13 @@
     "rpcName",
     "serviceName",
     "credentialsType",
-    "credentialInfo",
+    "credentialsInfo",
     "universeDomain",
     "request",
     "response",
     "metadata",
```

Comment:
Per the parent design doc, we should still have `rpcName`, right? (We're using it in the pending PR in the generator)

Reply:
Yes we should. I see it here it here in the list.